src/main/webapp/app/exercises/programming/shared/lifecycle/programming-exercise-lifecycle.component.ts

```diff
@@ -71,6 +77,7 @@ export class ProgrammingExerciseLifecycleComponent implements OnInit, OnChanges
             this.exercise.assessmentType = AssessmentType.AUTOMATIC;
             this.exercise.assessmentDueDate = undefined;
             this.exercise.allowComplaintsForAutomaticAssessments = false;
+            this.exercise.feedbackSuggestionsEnabled = false;
```

Comment:
Note for reviewers: When a programming exercise is changed to be `AUTOMATIC`, it does not support feedback suggestions anymore because there is no manual assessment where it would be shown.